packages/manager/modules/bm-server-components/src/general-info-tile/template.html

```diff
@@ -0,0 +1,155 @@
+<oui-tile
+    class="h-100"
+    heading="{{:: 'server_informations_general' | translate }}"
+>
+    <!-- name -->
+    <oui-tile-definition
+        term="{{:: 'server_configuration_displayname' | translate }}"
+        description="{{ $ctrl.server.displayName }}"
+    >
+        <oui-action-menu
+            data-ng-if="::$ctrl.showNameActions()"
+            data-compact
+            data-placement="end"
+        >
+            <oui-action-menu-item on-click="$ctrl.editName()">
+                <span
+                    data-translate="server_configuration_displayname_edit_button"
+                ></span>
+            </oui-action-menu-item>
+        </oui-action-menu>
+    </oui-tile-definition>
+    <!-- commercial range -->
+    <oui-tile-definition
+        term="{{:: 'server_configuration_description' | translate }}"
+    >
+        <oui-tile-description>
+            <span data-ng-bind="$ctrl.getFormatedCommercialRange()"> </span>
+        </oui-tile-description>
+    </oui-tile-definition>
+    <!-- upgrade -->
+    <div ng-transclude="serverUpgrade"></div>
+    <!-- system OS -->
+    <oui-tile-definition
+        term="{{:: 'server_configuration_distribution' | translate }}"
+    >
+        <oui-tile-description>
+            <span
+                data-ng-if="$ctrl.server.os"
+                data-ng-bind="$ctrl.server.os === 'byoi_32' ? ('server_configuration_distribution_byoi' | translate) : $ctrl.server.os"
+            ></span>
+            <span
+                class="oui-badge oui-badge_warning"
+                data-ng-if="!$ctrl.server.os"
+                data-translate="server_configuration_distribution_none"
+            >
+            </span>
+            <p
+                class="small"
+                data-ng-if="$ctrl.ola.isConfigured()"
+                data-translate="server_configuration_installation_no_public_network"
+            ></p>
+        </oui-tile-description>
+        <oui-action-menu
+            data-ng-if="!$ctrl.server.isExpired && !$ctrl.isHousingRange()"
+            data-compact
+            data-placement="end"
+            data-disabled="$ctrl.ola.isConfigured()"
+        >
+            <oui-action-menu-item
+                data-on-click="$ctrl.installOs('progress')"
```

Comment:
'progress' and 'choice' could be added to constants.

Reply:
Ideally yes. But not part of this refactoring work.